engines/postgres.py

```diff
@@ -130,5 +131,9 @@ def get_connection(self):
                              user=self.opts["user"],
                              password=self.opts["password"],
                              database=self.opts["database"])
-        conn.set_client_encoding('Latin1')
+
+        encoding = self.script.encoding
+        encoding_lookup = {'ISO-8859-1': 'LATIN1', 'utf-8': 'UTF8'}
```

Comment:
in case of postgres, which may differ from mysql or any other engine, we have https://www.postgresql.org/docs/current/static/multibyte.html.
We do not want to include all, I think Latin and utf are enough. 
For such cases I think we should include a comment saying that incase of any new encoding, update the encoding_lookup with the server's matching value